css/style.css

```diff
@@ -644,3 +646,178 @@ button:active {
   color: #42526e;
   margin: 0;
 }
+
+/** style popup model **/
+.model-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgb(193, 199, 208, 0.5);
+  opacity: 0;
+  pointer-events: none;
+}
+
+.model {
+  margin: 16px;
+  padding: 32px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: #fff;
+}
+
+.model-placeholder {
+  width: 100%;
+  height: 20vh;
+  background: url("../images/tech-image.png") no-repeat;
+  background-position: center center;
+  background-size: cover;
+  border: 1px solid #dfe1e6;
+  border-radius: 8px;
+  position: relative;
+}
+
+.icon-close {
+  width: 12px;
+  height: 12px;
+  position: absolute;
+  right: 22px;
+  top: 22px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  opacity: 1;
+}
+
+.project-title {
+  font-weight: 600;
```

Comment:
- [ ] On the popup window, your project titles (such as Multi-Post Stories Gain+Glory) need a `font-weight` of 700, not 600 according to the Figma.

Please, kindly correct that particular font styling attribute and your popup font stylings will be perfect! 💪

- [ ] On your readme file, kindly add all the sections missing from the required [Microverse readme template](https://github.com/microverseinc/readme-template).

Please, kindly remember all the sections are required, except for the "Tests" and "Live Demo" if not required by the project.

You are missing these 4 sections:

![image](https://user-images.githubusercontent.com/57421823/161963979-28acb26c-f38d-418e-9e31-23eaf6aea74a.png)


If you implement the remaining sections, your repo will look professional, and your hard work will be well documented! 💪